src/eradiate/scenes/atmosphere/_particle_layer.py

```diff
@@ -177,9 +177,18 @@ def _bottom_top_validator(self, attribute, value):
             converter=converters.load_dataset,
             validator=attr.validators.instance_of(xr.Dataset),
         ),
-        doc="Particle radiative property data set. If a path is passed, the "
-        "converter tries to open the corresponding file on the hard drive; if "
-        "this fails, it tries to load a resource from the data store.",
+        doc="""Particle radiative property data set. If a path is passed, the 
+converter tries to open the corresponding file on the hard drive; if
+this fails, it tries to load a resource from the data store.\n
+The data set must include the following data variables:\n
+   * ``sigma_t`` (``w``): volume extinction coefficient\n
+   * ``albedo`` (``w``): single-scattering albedo\n
+   * ``phase`` (``w``, ``mu``, ``i``, ``j``): scattering phase matrix\n
+(dimensions in brackets) and the following dimension coordinates: \n
+   * ``w``: radiation wavelength\n
+   * ``mu``: scattering angle cosine\n
+   * ``i``: scattering phase matrix row index\n
+   * ``j``: scattering phase matrix column index.""",
```

Comment:
I think the data guide is a better place for this. Then this docstring can refer to it.